packages/console/src/contexts/SignInExperienceContextProvider/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { noop } from '@silverhand/essentials';
+import { createContext, useMemo, useRef, useState } from 'react';
+
+type SignInExperienceContextType = {
+  isUploading: boolean;
+  cancelUpload?: () => void;
+  setIsUploading: (value: boolean) => void;
+  setCancelUpload: (cancelFunction?: () => void) => void;
+};
+
+type Props = {
+  readonly children?: React.ReactNode;
+};
+
+export const SignInExperienceContext = createContext<SignInExperienceContextType>({
```

Comment:
Only SIE page use this context, maybe move this file into related folder.

Reply:
Moved into sign-in-experience folder